### PR TITLE
85 bug all published ros topics are empty

### LINKED ISSUE #85: [Bug]: all "published" ROS Topics are empty
### Current Behavior

when starting the simulation with b5 run there are a lot of published topics (with `rostopic list` you can print them all).
But when i try to print the published messages to screen (with `rostopic echo <topic>`) there are none because i think theres something wrong with the clock.

Backlog:
```
(paf) imech037@auxme-imech037:~/PAF/paf22$ rostopic list

/carla/debug_marker
/carla/hero/Center/camera_info
/carla/hero/Center/image
/carla/hero/GPS
/carla/hero/IMU
/carla/hero/LIDAR
/carla/hero/OpenDRIVE
/carla/hero/RADAR
/carla/hero/Speed
/carla/hero/global_plan
/carla/hero/global_plan_gnss
/carla/hero/status
/carla/hero/vehicle_control_cmd
/carla/scenario/walker_control_cmd
/carla/status
/carla/world_info
/client_count
/clock
/connected_clients
/rosout
/rosout_agg
/tf

(paf) imech037@auxme-imech037:~/PAF/paf22$ rostopic echo /carla/hero/IMU
WARNING: no messages received and simulated time is active.
Is /clock being published?
^C
(paf) imech037@auxme-imech037:~/PAF/paf22$ rostopic echo /carla/hero/GPS
WARNING: no messages received and simulated time is active.
Is /clock being published?
^C
(paf) imech037@auxme-imech037:~/PAF/paf22$ rostopic echo /clock
WARNING: no messages received and simulated time is active.
Is /clock being published?
^C
(paf) imech037@auxme-imech037:~/PAF/paf22$ rosparam set use_sim_time false
(paf) imech037@auxme-imech037:~/PAF/paf22$ rostopic echo /clock
^C
(paf) imech037@auxme-imech037:~/PAF/paf22$ rostopic echo /carla/hero/GPS
```

### Expected Behavior

The topics should be filled with (sensor-) data to be read.

### How to reproduce the issue

(i was working on branch #71  [71-feature-navigation-data-research](https://github.com/ll7/paf22/tree/71-feature-navigation-data-research) for issue #71 should be up-to-date with main though)
- b5 run

when world and agent got started open another terminal:
- rostopic list (to see available <topic>) 
- rostopic echo <topic>

### PR DESCRIPTION
**Pull request by timdreier**
# Description

Add shortcuts for ROS commands.

Fixes #85 

## Time invested

1h 15m

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## Does this PR introduce a breaking change?

No.

## Most important changes

That the shortcuts are working, e.g. `b5 rostopic list`.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works (might be obsolete with CI later on)
- [x] New and existing unit tests pass locally with my changes (might be obsolete with CI later on)

### CODE REVIEW

File: build/tasks/ros.sh [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+ task:roscommand "rosstack ${@}"
+}
+
+task:rostopic() {
+ task:roscommand "rostopic ${@}"
+}
+
+task:rosversion() {
+ task:roscommand "rosversion ${@}"
+}
```
Comment (schwalga) [inline build/tasks/ros.sh:103]:
```suggestion
}
task:rqt_graph() {
  task:roscommand "rqt_graph ${@}"
}
```
The commands to open a GUI (rqt_topic, rqt_plot,...) would be them most useful in my opinion.

**Review by schwalga — CHANGES_REQUESTED** [review]
The ability to run ros commands directly sounds like a great idea to me.
I tested some of the commands. 
While simple command like `b5 rostopic list` work just fine, I can't get more advanced commands (`b5 rostopic echo /carla/status`) to run without an error.
I'm not sure if this is due to user error or if changes are needed?

I think it would be good idea to add a few more commands to enable tools like `rqt_topic`, because as you mentioned in the documentation, for more complex tasks it is easier to use `b5 shell`.

**Comment by timdreier** [comment]
Approved the requested changes and added the additional command to documentation.

Time invested: ~10 minutes

**Comment by schwalga** [comment]
All in all, a nice PR that adds very helpful functionality.
While I would have liked a few more commands (`rqt_graph`, `rqt_topic`, ...), I think it is time to merge this PR.

**Review by schwalga — APPROVED** [review]

**Pull request merged**